simple_history/models.py

```diff
@@ -265,10 +252,25 @@ def get_history_change_reason(self):
         else:
             history_id_field = models.AutoField(primary_key=True)
 
+        if self.history_change_reason_field:
+            # User specific field from init
+            history_change_reason_field = self.history_change_reason_field
+            history_change_reason_field.null = True
+        elif getattr(settings, 'SIMPLE_HISTORY_HISTORY_CHANGE_REASON_FIELD', False):
+            # User specific field from settings
+            history_change_reason_field = getattr(settings, 'SIMPLE_HISTORY_HISTORY_CHANGE_REASON_FIELD')
+            history_change_reason_field.null = True
+            history_change_reason_field.max_length = 500
```

Comment:
You should probably allow the `max_length` to be a setting as well. And leave it as 100 by default so that users don't have to makemigrations with the next release if they don't need this functionality